src/addons/Portal/Portal.js

```diff
@@ -363,6 +363,14 @@ class Portal extends Component {
     )
 
     this.portal = this.node.firstElementChild
+    // don't take focus away from portals that close on blur
+    if (!closeOnTriggerBlur) {
+      this.triggerNode = document.activeElement
```

Comment:
Nit: Could we use another name here?

If a portal is triggered programmatically, the `activeElement` will not necessarily be the `triggerNode`.  These are only the same when the user clicked on a trigger, such as a button.  How about something like `previousActiveElement` or similar?